### PR TITLE
Moves to use Bootstrap 4 as default

### PR DESCRIPTION
**Pull request by paganotoni**
Makes bootstrap 4 default for new apps.
Solves #1014

### CODE REVIEW

File: generators/newapp/templates/templates/application.html.tmpl [hunk 1]
```diff
@@ -5,10 +5,10 @@
     <meta charset="utf-8">
     <title>Buffalo - {{ .opts.Name.Title }}</title>
   {{- if not .opts.WithWebpack }}
-    {{- if eq .opts.Bootstrap 4 }}
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
-    {{- else }}
+    {{- if eq .opts.Bootstrap 3 }}
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+    {{- else }}
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
```
Comment (stanislas-m) [inline generators/newapp/templates/templates/application.html.tmpl]: This should be changed to the release version. :)

**Comment by paganotoni** [comment]
@stanislas-m changed.

**Comment by markbates** [comment]
@paganotoni looks like there are some merge conflicts that need resolving. can you resolve them?

**Comment by paganotoni** [comment]
Seems its working now (we may need to squash this commits)

**Pull request closed without merging**